test/lint/lint-shell.sh

```diff
@@ -51,7 +51,7 @@ do
     echo
     echo "$descriptor"
     # Use #!/bin/bash as gitian-builder/bin/gbuild does to complete a script.
```

Comment:
pleas update the comment as well

Reply:
updated comment

Reply:
https://github.com/bitcoin/bitcoin/pull/17751#discussion_r358908378